[2024-07-25T09:17:11.275+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-02T00:00:00+00:00 [queued]>
[2024-07-25T09:17:11.592+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-02T00:00:00+00:00 [queued]>
[2024-07-25T09:17:11.593+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-07-25T09:17:11.892+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): get_id_from_json> on 2023-04-02 00:00:00+00:00
[2024-07-25T09:17:12.048+0000] {standard_task_runner.py:57} INFO - Started process 540 to run task
[2024-07-25T09:17:12.088+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simple_ids_ingest', 'get_id_from_json', 'scheduled__2023-04-02T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/ids_ingest/ids_ingest_dag.py', '--cfg-path', '/tmp/tmp6d60xuei']
[2024-07-25T09:17:12.116+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask get_id_from_json
[2024-07-25T09:17:12.632+0000] {task_command.py:416} INFO - Running <TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-02T00:00:00+00:00 [running]> on host c7799c7b21e5
[2024-07-25T09:17:13.119+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ajay' AIRFLOW_CTX_DAG_ID='simple_ids_ingest' AIRFLOW_CTX_TASK_ID='get_id_from_json' AIRFLOW_CTX_EXECUTION_DATE='2023-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-02T00:00:00+00:00'
[2024-07-25T09:17:13.123+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ids_ingest/ids_ingest_dag.py", line 18, in get_ids_from_json
    with open(filename_json, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/files_to_test/github_events.json'
[2024-07-25T09:17:13.167+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=simple_ids_ingest, task_id=get_id_from_json, execution_date=20230402T000000, start_date=20240725T091711, end_date=20240725T091713
[2024-07-25T09:17:13.230+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 31 for task get_id_from_json ([Errno 2] No such file or directory: '/opt/airflow/files_to_test/github_events.json'; 540)
[2024-07-25T09:17:13.320+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-25T09:17:13.445+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-25T14:21:42.664+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-02T00:00:00+00:00 [queued]>
[2024-07-25T14:21:42.906+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-02T00:00:00+00:00 [queued]>
[2024-07-25T14:21:42.922+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-07-25T14:21:43.129+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): get_id_from_json> on 2023-04-02 00:00:00+00:00
[2024-07-25T14:21:43.231+0000] {standard_task_runner.py:57} INFO - Started process 3340 to run task
[2024-07-25T14:21:43.363+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simple_ids_ingest', 'get_id_from_json', 'scheduled__2023-04-02T00:00:00+00:00', '--job-id', '339', '--raw', '--subdir', 'DAGS_FOLDER/ids_ingest/ids_ingest_dag.py', '--cfg-path', '/tmp/tmp6iftwpb8']
[2024-07-25T14:21:43.376+0000] {standard_task_runner.py:85} INFO - Job 339: Subtask get_id_from_json
[2024-07-25T14:21:44.147+0000] {task_command.py:416} INFO - Running <TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-02T00:00:00+00:00 [running]> on host b7334bec8397
[2024-07-25T14:21:45.864+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ajay' AIRFLOW_CTX_DAG_ID='simple_ids_ingest' AIRFLOW_CTX_TASK_ID='get_id_from_json' AIRFLOW_CTX_EXECUTION_DATE='2023-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-02T00:00:00+00:00'
[2024-07-25T14:21:45.981+0000] {logging_mixin.py:154} INFO - ['25208138097', '25208138110', '25208138076', '25208138082', '25208138077', '25208138087', '25208138078', '25208138069', '25208138072', '25208138080', '25208138070', '25208138067', '25208138068', '25208138066', '25208138047', '25208138051', '25208138053', '25208138060', '25208138071', '25208138061', '25208138030', '25208138044', '25208138043', '25208138038', '25208138032', '25208138018', '25208138016', '25208138012', '25208138008', '25208137998']
[2024-07-25T14:21:46.004+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-07-25T14:21:46.158+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_ids_ingest, task_id=get_id_from_json, execution_date=20230402T000000, start_date=20240725T142142, end_date=20240725T142146
[2024-07-25T14:21:46.534+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-25T14:21:46.904+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
